### PR TITLE
Added unit test to check whether channels that contain data are ident…

### PR DESCRIPTION
**Pull request by nikhilchandra**
…ified correctly when loading a PL2 file.

### CODE REVIEW

**Comment by nikhilchandra** [comment]
This may have failed because it depends on the acceptance of my previous pull request. Sorry if I should have waited until the other one was accepted.

https://github.com/NeuralEnsemble/python-neo/pull/1551

File: neo/test/rawiotest/test_plexon2rawio.py [hunk 1]
```diff
@@ -25,8 +26,45 @@ class TestPlexon2RawIO(
...
+                        ]
+    
 
+    def test_check_enabled_flags(self):
+        """
+        This test loads a 1-minute PL2 file with 16 channels' each
+        of field potential (FP), and spike (SPK) data. The channels 
+        cycle through 4 possible combinations of m_ChannelEnabled 
+        and m_ChannelRecordingEnabled - (True, True), (True, False), 
+        (False, True), and (False, False). With 16 channels for each
```
Comment (samuelgarcia) [inline neo/test/rawiotest/test_plexon2rawio.py:40]: How can a channel be not enabled but recorded ?

**Comment by samuelgarcia** [comment]
Merci beaucoup

File: neo/test/rawiotest/test_plexon2rawio.py (hunk 1 above)
Comment (zm711) [reply neo/test/rawiotest/test_plexon2rawio.py:40]: https://github.com/NeuralEnsemble/python-neo/pull/1546#issue-2505718693

See comment here Sam :)

**Comment by zm711** [comment]
> This may have failed because it depends on the acceptance of my previous pull request. Sorry if I should have waited until the other one was accepted.

That's fine. We can review that one first and merge this one later. Not a biggie.

**Review by zm711 — APPROVED** [review]
Thanks for the test @nikhilchandra :)

**Comment by zm711** [comment]
I'm going to update the branch and then if tests pass after that I'll merge.

**Comment by nikhilchandra** [comment]
Thanks @zm711. I was trying to get my Linux box set up to run the test locally, sorry for the delay in response. Glad everything seems to work now.

**Pull request merged**